holoviews/plotting/mpl/element.py

```diff
@@ -15,6 +16,7 @@
 from ..util import dynamic_update
 from .plot import MPLPlot
 from .util import wrap_formatter
+from distutils.version import LooseVersion
```

Comment:
I'm not sure we should assume distutils is available (which might explain the current test failures).